src/main/drivers/pwm_output.c

```diff
@@ -40,6 +40,11 @@ static pwmCompleteWriteFuncPtr pwmCompleteWritePtr = NULL;
 static pwmOutputPort_t servos[MAX_SUPPORTED_SERVOS];
 #endif
 
+#ifdef BEEPER_PWM
+static pwmOutputPort_t beeperPwm;
+static uint16_t freqBeep=BEEPER_PWM_FREQ;
```

Comment:
This shouldn't be set here, but as default for `frequency`: src/main/io/beeper.chttps://github.com/Faduf/betaflight/blob/25d7d1e66966aea6e6627d5635b2f6b62edef86a/src/main/io/beeper.c#L63